content/docs/how-we-work/task-estimation/estimation-process.md

```diff
@@ -14,6 +14,6 @@ We ensure that the necessary grooming is done, and the required details are adde
 
 ## Estimation Templates
 
-Use the [master estimate sheet template](https://docs.google.com/spreadsheets/d/1wXAFXsSbiQfaMSCMGr-7GvWHYh2O9fc5mhUOQgJsos8/edit?gid=683107236#gid=683107236) and follow the instructions given in the sheet. The team can raise a request or suggest feedback or invite a call to discuss any challenges, which will help revise the estimation sheet iteratively. Add estimates in internal and bug tickets as well and get those reviewed by the TL. Add a link and a screenshot of the estimation sheet on the respective ticket. The screenshot is important as it will help us identify any changes done later to the estimations. We also use the [CDM ticket estimation template](https://docs.google.com/spreadsheets/d/1NExM6FlLe0kWwhFNPp4RBi8E1D7iixRQHPVriP22B1Y/edit?gid=0#gid=0) only for ticket-level estimation.
+Use the [master estimate sheet template](https://docs.google.com/spreadsheets/d/1wXAFXsSbiQfaMSCMGr-7GvWHYh2O9fc5mhUOQgJsos8/edit?gid=683107236#gid=683107236) and follow the instructions given in the sheet. The team can raise a request or suggest feedback or invite a call to discuss any challenges, which will help revise the estimation sheet iteratively. Add estimates in internal and bug tickets as well and get those reviewed by the TL. Add a link and a screenshot of the estimation sheet on the respective ticket. The screenshot is important as it will help us identify any changes done later to the estimations. We also use the [CDM ticket estimation template](https://docs.google.com/spreadsheets/d/1NExM6FlLe0kWwhFNPp4RBi8E1D7iixRQHPVriP22B1Y/edit?gid=0#gid=0) only for ticket-level estimation. If there are tickets that might require more than a week of effort, use the [master estimate sheet template](https://docs.google.com/spreadsheets/d/1wXAFXsSbiQfaMSCMGr-7GvWHYh2O9fc5mhUOQgJsos8/edit?gid=683107236#gid=683107236).
```

Comment:
@Yemaneberhan-Lemma can we merge the week more point in 1st line itself. Reading it now it looks like repetition. 

We can change 1st line to be 

Use the [master estimate sheet template](https://docs.google.com/spreadsheets/d/1wXAFXsSbiQfaMSCMGr-7GvWHYh2O9fc5mhUOQgJsos8/edit?gid=683107236#gid=683107236) and follow the instructions given in the sheet for tickets that might be a CR requiring more than a weeks' effort.